common/src/main/java/com/wynntils/core/features/overlays/BarOverlay.java

```diff
@@ -78,6 +82,8 @@ protected void render(
 
         String textValue = templateCache.a();
         CappedValue value = valueOrError.getValue();
+        if (value.equals(CappedValue.EMPTY)) return;
```

Comment:
I don't think I agree with this? This should be a config, but we already have a way of setting visibility with templates.

Reply:
So what do you want to display for something that returns an EMPTY capped value? Note that this is essentially "impossible", it is 0/0. No normal data will have this, so it is used to signal essentially "n/a".

Reply:
Also note that this is not so much a config, as that a bar that has no value to display should not be drawn. As soon as you get proper data (max != 0), then it should of course be drawn again.

Reply:
Okay, let's go with this and I will see if I want to change it later.